api/http/router.go

```diff
@@ -34,29 +41,110 @@ const (
 
 	APIURLManagement = "/api/management/v1/azure-iot-manager"
 
-	APIURLSettings = "/settings"
+	APIURLSettings      = "/settings"
+	APIURLDevice        = "/device/:id"
+	APIURLDeviceTwin    = "/device/:id/twin"
+	APIURLDeviceModules = "/device/:id/modules"
```

Comment:
Dummy question: `/device` or `/devices`?

Reply:
I prefer singular `/device` but we mostly use plural `/devices`, I don't know which one's "correct".

Reply:
Changed it to `/devices/:id`